packages/database/src/modelClasses/Task.js

```diff
@@ -55,6 +55,17 @@ export class TaskRecord extends DatabaseRecord {
   async systemComments() {
     return this.otherModels.taskComment.find({ task_id: this.id, type: 'system' });
   }
+
+  async addComment(message, userId, type = 'user') {
```

Comment:
I'm like that you are adding extra methods to the model classes. It's something that we haven't done enough in the past in tupaia